kaggle_plantpatho/data.py

```diff
@@ -15,6 +16,11 @@
 from torchvision import transforms as T
 from torchvision.transforms import InterpolationMode
 
+try:
+    from torchsampler import ImbalancedDatasetSampler
```

Comment:
why don't we enable by default?

Reply:
because it is not packaged on PyPI yet so if a user installs this package this sampler is missing